tests/integration_tests/configs/upgrade-test-package.nix

```diff
@@ -4,9 +4,9 @@ let
   released = pkgs.buildGo118Module rec {
     name = "ethermintd";
     # the commit before https://github.com/evmos/ethermint/pull/943
-    src = fetchEthermint "f21592ebfe74da7590eb42ed926dae970b2a9a3f";
+    src = fetchEthermint "8866ae0ffd67a104e9d1cf4e50fba8391dda6c45";
```

Comment:
can we do this automatically instead?

Reply:
might take longer time to build if under hot development, and some cli need code change

Reply:
> might take longer time to build if under hot development, and some cli need code change

Can you create an issue so that we can track this?

Reply:
I'm not good at doc, see if need any update https://github.com/evmos/ethermint/issues/1514